### ISSUE TITLE
Atum / Media Manager / Update Joomla: Keyboard shortcuts have a time limit for use

### ISSUE THREAD

**Issue by mbeganyi-a11y** [joomla/joomla-cms#48420]
### Steps to reproduce the issue

**NOTE**: Accessibility backlog issue and work-in-progress. This issue has been opened to account for all the issues flagged in the initial [Milestone 1a Audit Report](https://accessibility-project.joomla.org/results/Milestone-1a/milestone1/deliverable1/wcag-report). If this is a duplicate issue, it will be closed by the end of September 2026. 

Milestone 1a   finding ID | M1a-005
-- | --
Backlog   priority | Low
Severity —   carried from Milestone 1a | Minor
Implementation   effort | Low
Target   milestone(s) | MS7a, MS7b,   MS7c, MS8a, MS9a, MS10
WCAG 2.2   success criterion | 2.2.1 Timing   Adjustable
Responsibility   tag(s) from report | Development
Affected   Joomla area/component | https://accessibility-project.joomla.org/results/Milestone-1a/milestone1/deliverable1/wcag-report#13b5fcb8-9e38-45e1-b404-e6a6f0adae96
Affected user   journey(s) | 01 – Backend   Template (Atum); 05 – MFA Setup - Method / Methods Views; 06 – Media Manager   (com_media); 07 – Installer Wizard (com_installer/Install); 08 – Backend List   View Pattern; 09 – Backend Edit Form Pattern; 11 – Backend Options including   Permissions; 12 – Update Joomla
Source   report/reference | Milestone 1a   WCAG audit report
STF reporting   reference | To be   assigned
Issue owner | To be   assigned
Status | Triage

<p class=MsoNormal>Audit finding<o:p></o:p></p>

<p class=MsoNormal>Keyboard shortcuts time limit<o:p></o:p></p>

<p class=MsoNormal>User impact<o:p></o:p></p>

<p class=MsoNormal>Keyboard-only users and many assistive-technology users may
be blocked from operating the interface or completing the task due to a constriction
of time on keyboard shortcuts<o:p></o:p></p>

<p class=MsoNormal>Remediation plan<o:p></o:p></p>

<p class=MsoNormal>Refactor the interaction so it is fully keyboard operable
with a time limit of next input up to 10 times the initial length<o:p></o:p></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'>Steps to reproduce</b><o:p></o:p></p>

<p class=MsoNormal>1. Open the Global Configuration page in the Backend options.<o:p></o:p></p>

<p class=MsoNormal>2. Trigger the reported single-key shortcut.<o:p></o:p></p>

<p class=MsoNormal>3. Repeat with focus on other controls and content that aren't text-entry fields.<o:p></o:p></p>

<p class=MsoNormal>4. Observe that timing is very low to use the next key in
the shortcut sequence.<o:p></o:p></p>

</body>

</html>

### Version

6.1

### Expected result

More time is allowed to trigger the keyboard shortcut sequence.

### Actual result

When using the J plus other key shortcuts, there is a time limit between hitting one key and the next. This sequence is relatively short and some users with disabilities will find it difficult to press two buttons in short sequence.

### What device were you using? (eg, phone, laptop)

Desktop web

### What web browser were you using? (eg, Chrome, Safari)

Chrome, Firefox, Edge, Safari

**Comment by brianteeman** [comment]
In the steps to reproduce it says

> 2. Place focus in a text-entry field.

> 3. Trigger the reported single-key shortcut.

There are no single-key shortcuts and the two key J+somekey shortcuts do **not** work at all when you are in a text entry field

**Comment by chmst** [comment]
J + S does not work in a text field. Otherwise it would be impossible to write something like "I am learning JS "

**Comment by mbeganyi-a11y** [comment]
Thanks for the feedback. I've changed the information.
